src/config/configuration.ts

```diff
@@ -22,8 +22,8 @@ if (!isProduction) {
 
 const envConfig = isProduction ? production : development;
 const config = {
-  LOW_CAP_TEXT: 'approximately $1,000 denominated in POL',
-  HIGH_CAP_TEXT: 'approximately $15,000 denominated in POL',
+  LOW_CAP_TEXT: '1,356 POL',
```

Comment:
you need to get this value from BE, because they are changing, I think we need to discause about the API I should provide to you